<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smarthomeDialog</class>
 <widget class="QDialog" name="smarthomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>smarthomeDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>QT smart home with tcp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ipLine"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>port </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="PortspinBox"/>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>relay 1</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="RelayONpushButton">
          <property name="text">
           <string>on</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="RelayOffpushButton">
          <property name="text">
           <string>off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="AllOnpushButton">
       <property name="text">
        <string>ALL on</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>relay 2</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="RelayOn2pushButton">
          <property name="text">
           <string>on</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="RelayOff2pushButton">
          <property name="text">
           <string>off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="AllOffpushButton">
       <property name="text">
        <string>ALL off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
